packages/core/theme-module/index.js

```diff
@@ -37,14 +45,19 @@ module.exports = async function DefaultThemeModule(moduleOptions) {
   };
 
   log.info('Adding theme files...');
+  const themeDirectoriesPaths = [];
+  const copyThemeDirectoriesPromises = [];
+
+  for (const directory of getDirectoriesList(this.options.rootDir)) {
+    if (!omittedDirectories.includes(directory)) {
+      const absolutePath = path.join(this.options.rootDir, directory);
+      themeDirectoriesPaths.push(absolutePath);
+      copyThemeDirectoriesPromises.push(copyThemeFiles(absolutePath));
+    }
+  }
```

Comment:
**code organization** Grouping highlited code into a function that returns array of paths (like themeFiles) will make this piece more readable and keep all the code related to this particular action in one place (easier to move or remove).

Also in that case `filter` seems much more natural than `for...of`. 

Try to find similar patterns in the file and follow them 
```
const themeFiles = getAllFilesFromDir(baseThemeDir).filter(file => !file.includes(path.sep + 'static' + path.sep));
```